creating new nx workspace with preset=qwik-nx using pnpx throws an error

**Describe the bug**

Creating a new workspace with the qwik-nx preset throws (I'm using windows 10)

**To Reproduce**

Run ` pnpx create-nx-workspace --preset qwik-nx`


**Expected behavior**
To complete with no errors

**Screenshots**
```
node:events:491
      throw er; // Unhandled 'error' event
      ^

Error: spawn pnpm ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
Emitted 'error' event on ChildProcess instance at:
    at ChildProcess._handle.onexit (node:internal/child_process:289:12)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn pnpm',
  path: 'pnpm',
  spawnargs: [
    'exec',
    'nx',
    '--nxWorkspaceRoot="C:/dev/workspace/open-source"',
    'g',
    'qwik-nx:preset',
    '--no-a',
    '--no-g',
    '--$0=C:\\Users\\MyUser\\AppData\\Local\\pnpm\\store\\v3\\tmp\\dlx-19560\\node_modules\\create-nx-workspace\\bin\\index.js',
    '--packageManager=pnpm',
    '--defaultBase=main'
  ]
}

Node.js v18.7.0
 ERROR  Command failed with exit code 1: C:\Users\MyUser\AppData\Local\pnpm\store\v3\tmp\dlx-19560\node_modules\.bin\create-nx-workspace.CMD --preset qwik-nx

pnpm: Command failed with exit code 1: C:\Users\MyUser\AppData\Local\pnpm\store\v3\tmp\dlx-19560\node_modules\.bin\create-nx-workspace.CMD --preset qwik-nx
    at makeError (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:22852:17)
    at handlePromise (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:23423:33)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.handler [as dlx] (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:202440:7)
    at async C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:209882:21
    at async main (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:209853:34)
    at async runPnpm (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:210079:5)
    at async C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:210071:7

```

**Desktop (please complete the following information):**

- OS: Windows 10

On macOS 10.15.6 it fails with the following
```
 >  NX   Cannot find module '@nxkit/playwright'

   Require stack:
   - /Users/dmitriy/projects/test/cuterepo/node_modules/qwik-nx/src/generators/application/lib/add-e2e-project.js
   - /Users/dmitriy/projects/test/cuterepo/node_modules/qwik-nx/src/generators/application/generator.js
   - /Users/dmitriy/projects/test/cuterepo/node_modules/qwik-nx/src/generators/preset/generator.js
   - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/src/config/workspaces.js
   - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/src/config/configuration.js
   - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/src/command-line/generate.js
   - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/src/command-line/nx-commands.js
   - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/bin/init-local.js
   - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/bin/nx.js
   Pass --verbose to see the stacktrace.


 >  NX   Failed to apply preset: qwik-nx
```

> On macOS 10.15.6 it fails with the following
> 
> ```
>  >  NX   Cannot find module '@nxkit/playwright'
> 
>    Require stack:
>    - /Users/dmitriy/projects/test/cuterepo/node_modules/qwik-nx/src/generators/application/lib/add-e2e-project.js
>    - /Users/dmitriy/projects/test/cuterepo/node_modules/qwik-nx/src/generators/application/generator.js
>    - /Users/dmitriy/projects/test/cuterepo/node_modules/qwik-nx/src/generators/preset/generator.js
>    - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/src/config/workspaces.js
>    - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/src/config/configuration.js
>    - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/src/command-line/generate.js
>    - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/src/command-line/nx-commands.js
>    - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/bin/init-local.js
>    - /Users/dmitriy/projects/test/cuterepo/node_modules/nx/bin/nx.js
>    Pass --verbose to see the stacktrace.
> 
> 
>  >  NX   Failed to apply preset: qwik-nx
> ```

I'm seeing the same error ... This is what I have

node v16.19.0
create-nx-workspace@15.6.3

@dmitry-stepanenko  If I call the generator twice, the second call works as expected. I'll take a deeper look later

Steps:
```
npx create-nx-workspace@latest nx-qwik-test (used the integrated scheme)
npm install --save qwik-nx
npx nx g qwik-nx:app app
```

Logs:
```
npx nx g qwik-nx:app app --verbose

>  NX  Generating qwik-nx:application

✔ Which stylesheet format would you like to use? · css
added 1 package, removed 32 packages, and audited 350 packages in 796ms

31 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

 >  NX   Cannot find module '@nxkit/playwright'

   Require stack:
   - /private/tmp/nx-qwik-test/node_modules/qwik-nx/src/generators/e2e-project/generator.js
   - /private/tmp/nx-qwik-test/node_modules/qwik-nx/src/generators/application/generator.js
   - /private/tmp/nx-qwik-test/node_modules/nx/src/config/workspaces.js
   - /private/tmp/nx-qwik-test/node_modules/nx/src/config/configuration.js
   - /private/tmp/nx-qwik-test/node_modules/nx/src/command-line/generate.js
   - /private/tmp/nx-qwik-test/node_modules/nx/src/command-line/nx-commands.js
   - /private/tmp/nx-qwik-test/node_modules/nx/bin/init-local.js
   - /private/tmp/nx-qwik-test/node_modules/nx/bin/nx.js

Error: Cannot find module '@nxkit/playwright'
Require stack:
- /private/tmp/nx-qwik-test/node_modules/qwik-nx/src/generators/e2e-project/generator.js
- /private/tmp/nx-qwik-test/node_modules/qwik-nx/src/generators/application/generator.js
- /private/tmp/nx-qwik-test/node_modules/nx/src/config/workspaces.js
- /private/tmp/nx-qwik-test/node_modules/nx/src/config/configuration.js
- /private/tmp/nx-qwik-test/node_modules/nx/src/command-line/generate.js
- /private/tmp/nx-qwik-test/node_modules/nx/src/command-line/nx-commands.js
- /private/tmp/nx-qwik-test/node_modules/nx/bin/init-local.js
- /private/tmp/nx-qwik-test/node_modules/nx/bin/nx.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1039:15)
    at Module._load (node:internal/modules/cjs/loader:885:27)
    at Module.require (node:internal/modules/cjs/loader:1105:19)
    at require (/private/tmp/nx-qwik-test/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at /private/tmp/nx-qwik-test/node_modules/qwik-nx/src/generators/e2e-project/generator.js:35:73
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Nx Report:
```
npx nx report

 >  NX   Report complete - copy this into the issue template

   Node : 18.13.0
   OS   : darwin x64
   npm  : 8.19.3

   nx : 15.6.3
   @nrwl/angular : Not Found
   @nrwl/cypress : Not Found
   @nrwl/detox : Not Found
   @nrwl/devkit : 15.6.3
   @nrwl/esbuild : Not Found
   @nrwl/eslint-plugin-nx : 15.6.3
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : Not Found
   @nrwl/js : 15.6.1
   @nrwl/linter : 15.6.3
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/rollup : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : Not Found
   @nrwl/web : Not Found
   @nrwl/webpack : Not Found
   @nrwl/workspace : 15.6.3
   @nrwl/vite : 15.6.1
   typescript : 4.8.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
   	 qwik-nx: 0.9.0
   	 @nxkit/playwright: 2.1.0
``

--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
     "@nrwl/nx-plugin": "15.6.1",
     "@nrwl/vite": "15.6.1",
     "@nrwl/workspace": "15.6.1",
-    "@nxkit/playwright": "2.0.1",
+    "@nxkit/playwright": "^2.1.1",
     "@swc-node/register": "^1.4.2",
     "@swc/cli": "~0.1.55",
     "@swc/core": "^1.2.173",

--- a/packages/qwik-nx/src/utils/versions.ts
+++ b/packages/qwik-nx/src/utils/versions.ts
@@ -17,7 +17,7 @@ export const postcssVersion = '8.4.16';
 export const tailwindcssVersion = '3.1.8';
 
 // nxkit packages
-export const nxKitVersion = '^2.0.1';
+export const nxKitVersion = '^2.1.1';
 
 // cloudflare-pages integration
 export const wranglerVersion = 'latest';

--- a/tsconfig.base.json
+++ b/tsconfig.base.json
@@ -15,8 +15,8 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "qwik-nx": ["packages/qwik-nx/src/index.ts"],
-      "@qwikifiers/e2e/utils": ["e2e/utils"]
+      "@qwikifiers/e2e/utils": ["e2e/utils"],
+      "qwik-nx": ["packages/qwik-nx/src/index.ts"]
     }
   },
   "exclude": ["node_modules", "tmp"]

